funnel/forms/profile.py

```diff
@@ -20,7 +20,8 @@ class EditProfileForm(forms.Form):
     description = forms.MarkdownField(__("Welcome message"),
         validators=[forms.validators.DataRequired(_(u"Please write a message for the landing page"))],
         description=__("This welcome message will be shown on the landing page."))
-    logo_url = forms.URLField(__("Logo URL"), description=__("Profile logo"),
+    logo_url = forms.URLField(__("Logo URL"),
+        description=__("Optional — Profile logo will be shown on project landing page"),
```

Comment:
Profile landing page or project landing page?

Reply:
The dashes need to be n-dashes. (Also below.) That will require either using a unicode string -- `u"Optional – The profile logo will be shown on the profile’s landing page"` -- or using `from __future__ import unicode_literals` at the top of the file.